### ISSUE TITLE
fix: 🤔 CTA button color

### ISSUE THREAD

**Issue by coraliefeil** [solid-design-system/solid#809]
## Current behavior
sd-button/cta background color "background.accent" "#2D9D01"
-> Not accessible! Insufficient color contrast.

## Expected behavior
sd-button/cta background color "background.accent-500" "#25880E"
[Hand-off](https://www.figma.com/file/fPGhgNZv98U4H69Gu2tlWi/Button?type=design&node-id=2572-3119&mode=design&t=o7HDPqPHKNCGc3nA-11)

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [x] Implementation works successfully on `feature` branch
